tests/ibc-hooks/ibc_middleware_test.go

```diff
@@ -48,9 +48,6 @@ type HooksTestSuite struct {
 	pathAB *ibctesting.Path
 	pathAC *ibctesting.Path
 	pathBC *ibctesting.Path
-	pathBA *ibctesting.Path
-	pathCA *ibctesting.Path
-	pathCB *ibctesting.Path
```

Comment:
imo it might make sense to leave these, just in case they might be useful in the future